modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -141,35 +153,34 @@ External mount points are not part of the automatic migration. See the following
 * To migrate any external mount, the https://www.kiteworks.com/enterprise-connect/[Kiteworks Enterprise Connect] license is required.
 * If an external mount is encrypted, it must be decrypted first.
 * Follow the Kiteworks instructions to (re)connect an external mount.
-* Federated shares need by their nature individual treatment, no general advice can be given.
+* Federated shares need, by their nature, individual treatment, no general advice can be given.
 
 For ease of migrating external mounts, the admin should:
 
 * For admin created mounts, make a list of mounts with their settings and their sharing configuration.
-* For user created external mounts, the administrator is responsible to instruct users how to migrate including how to re-setup sharing.
+* For user created external mounts, the administrator is responsible to instruct users how to migrate, including how to re-setup sharing.
 
 === Kiteworks
 
-* As a major prerequisite, the Kiteworks instance *must* be running on the "Venice" release or higher. If this requirement is not met, migration can not be started.
+* As a major prerequisite, the Kiteworks instance *must* be running on the "Venice" release or higher. If this requirement is not met, migration cannot be started.
 
 * You need to login into the Kiteworks appliance as role *System Admin*.
 // The kiteworks satellite service must be activated and available to the system admin user account.
 
 * The Kiteworks system must provide sufficient disk space for the data to be migrated. The ownCloud xref:migration-verification[occ migrate:to-kiteworks:verify] step will report the estimated disk space needed.
 
-* Assure sufficient quota settings in the Kiteworks user profiles.
+* Ensure sufficient quota settings in the Kiteworks user profiles.
 
 * If it is planned to integrate Kiteworks into LDAP:
 +
 --
-IMPORTANT: We recommend to have the Kiteworks PCN connected and configured to an LDAP server _before_  starting the migration. This will avoid conflicting user entries that will exist in the local database additionally to the LDAP server connected.
-// New user accounts will be created during the migration, as needed. Existing user accounts will be used.
+IMPORTANT: We recommend having the Kiteworks PCN connected and configured to an LDAP server _before_  starting the migration. This will avoid conflicting user entries that will exist in the local database additionally to the LDAP server connected.
```

Comment:
This change to line 165-177 is good, but was not in my original PR.
And we don't want to lose old line 166.